orttraining/orttraining/test/python/orttraining_run_bert_pretrain.py

```diff
@@ -29,6 +30,7 @@
 import onnxruntime as ort
 from onnxruntime.training import amp, optim, orttrainer
 from onnxruntime.training.optim import PolyWarmupLRScheduler, LinearWarmupLRScheduler
+from onnxruntime.training.checkpoint import experimental_save_checkpoint, _list_checkpoint_files, _CombineZeroCheckpoint
```

Comment:
Are `_list_checkpoint_files, _CombineZeroCheckpoint` expected to be used in the public API for any scenario or this is just for unit testing purposes?

Reply:
The users should have the option to aggregate checkpoints offline, as there have been practical scenarios where the model was too large causing delays for checkpoint aggregation to be done within a training run. The design can be refined to expose this functionality in a better way of course.